package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "build": "NODE_ENV=production babel-node ./node_modules/webpack/bin/webpack --config webpack/prod.config.js",
-    "start": "NODE_ENV=development babel-node ./node_modules/webpack-dev-server/bin/webpack-dev-server --config webpack/dev.config.js",
+    "start": "NODE_ENV=development babel-node ./node_modules/webpack-dev-server/bin/webpack-dev-server --config webpack/dev.config.js --open",
```

Comment:
What is the open flag for?

Reply:
The open flag starts the app automatically in the browser without typing URL in browser as create react app does when you run the app.